### ISSUE TITLE
Lights don't work with the latest version

### ISSUE THREAD

**Issue by Dehumanizer77** [RobHofmann/HomeAssistant-GreeClimateComponent#141]
**Describe the bug**
Lights boolean input has stopped working after upgrade to 2.4.0, more specifically it looks like it's read only. When I turn on display via AC remote, the input_boolean changes to true, if I turn it off, it changes to false, but when I change the input manually via HA, it does not change the actual state of the display. It was working perfectly before. 

**Configuration**
- platform: gree
  name: Serverovňa
  host: 172.16.0.101
  port: 7000
  mac: 'f4:91:1e:62:7a:0c'
  target_temp_step: 1
  temp_sensor: sensor.serverovna_teplota
  lights: input_boolean.ac_serverovna_display

**Platform:**
 - OS: HassOS 10.2, HA Core 6.1
 - Version [e.g. 0.92.1]

**Comment by codo70** [comment]
Same with me also 8 Degrees option and external Sensor

**Comment by poisondima** [comment]
After updating to version 2.4.0 the following settings stopped working:
```
      temp_sensor: sensor.temp_0_temperature
      lights: input_boolean.gree_lights
      xfan: input_boolean.gree_xfan
      health: input_boolean.gree_health
      sleep: input_boolean.gree_sleep
      powersave: input_boolean.gree_powersave
```
Rolled back to version 2.3.2 and everything works again!
p.s. HA version 2023.5.4

**Comment by smarecki** [comment]
Same problem with lights.

**Comment by RobHofmann** [comment]
Should be fixed with 2.4.1.

Let me know.

**Comment by Dehumanizer77** [comment]
works fine! 👍
